content/03.prior-knowledge.md

```diff
@@ -0,0 +1,22 @@
+In most cases of rare disease, due to the inherent nature of being less defined, the symptoms as well as any underlying biology comes with a reasonable amount of uncertainty (or "label-noise") leading to a _silver standard_ dataset[@doi:10.1109/TNNLS.2013.2292894, @doi:10.1093/jamia/ocw028].
+In such datasets, ensemble learning or _cascade learning_ is used where multiple methods leveraging distinct underlying assumptions are used in tandem and augmented with algorithms like AdaBoost (_boosting_) to capture stable patterns existing in the _silver standard_ data and reduce uncertainty [@doi:10.1109/CVPR.2001.990537, @doi:10.1007/978-3-540-75175-5_16, @doi:10.1109/ICPR.2004.1334680].
+A variation of _cascade learning_ implemented to identify rare disease patients from electronic health records from the general population utilized independent steps for feature extraction (using word2vec [@arXiv:1301.3781v3]), preliminary prediction (ensemble of decision trees with penalization for excessive tree-depth), and prediction refinement (using similarity of data points to resolve sample labels) [@pmid:30815073].
+This cascade learner benefited from the independence of the feature extraction step and the prediction refinement step from the preliminary classification of the labeled dataset to find stable patterns and perform better than other ensemble methods when implemented on this silver standard dataset.
+
+In datasets with multiple classes, most cascade classifiers follow _one-classifier-at-a-time_ approach where algorithms at each level predict all classes involved.
+But instances where the need for high prediction accuracy for one class outweighs other classes, further modification of the cascade learning efforts is required.
+An example of such modification was implemented for triaging psychiatric patients where the identification of one class of psychiatric patients ("severe") far outweighed the need for optimized overall classification accuracy [@pmid:30380082].
+Due to the requirements of the problem, a _one-class-at-a-time_ cascade learning approach was adopted, where at each stage a binary classifier was used to predict a specific class against all others.
+The final model implemented all models together each identifying one class sequentially and the union of the predictions of all the different models as the final prediction.
+The cascade classifiers using the _one-class-at-a-time_ approach were found to perform better than multi-class ensemble classifiers in most cases.
+
+Thus ensemble learning can be helpful in producing stable predictions from rare disease data, but the choice of using _bagging_, _boosting_, independent algorithmic steps, or _one-class-at-a-time_ approach depends on the nature of the prediction question.
+The limited success of the _bagging_ approach in rare disease has encouraged various methodological modifications as discussed above.
```

Comment:
```suggestion
The limited success of the bagging approach in rare disease has encouraged various methodological modifications as discussed above.
```

Reply:
This sentence seems out of place, where as line 21 seems to be a better concluding sentence.

Reply:
revisiting this